Instagram Reels Bot/Helpers/IGComponentBuilder.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using Discord;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Instagram_Reels_Bot.Helpers
+{
+    internal class IGComponentBuilder
+    {
+        /// <summary>
+        /// The Response from the Instagram processor.
+        /// </summary>
+        private InstagramProcessorResponse Response;
+        private string Requester;

```

Comment:
No need for `Requester` here. Not really going to need their name on a button or anything. It's more important for embeds.

Reply:
Fixed in https://github.com/bman46/InstagramEmbedDiscordBot/pull/23/commits/a657066f90223d9063c82384e542d82ae7fc9b79